docs/learn/token.md

```diff
@@ -3,39 +3,66 @@ id: token
 title: Tokens
 ---
 
-Tokens is an important feature on Archethic. 
+Tokens is an important feature on Archethic.
 Nowadays, our world is transforming towards digitalisation and tokenization with new use cases and trends (royalties, collection, proof of ownerships, etc.)
 
 Archethic allows users to create and transaction with an unlimited-defined (custom) token natively.
 
-## Native 
+## Native
 
 Archethic tokens are considered as native as for developers, there is no need to create smart contract to mint or transfer tokens.
 
 The transaction's structure supports - by design, asn extension of the ledger model (more than UCO-only transaction)
 
 All the layer of complexity have been removed to make them efficient and performant.
 
-Because of this, the transfer of tokens is as simple as a transfer of UCO (just UTXO), making it fast and cheap. 
+Because of this, the transfer of tokens is as simple as a transfer of UCO (just UTXO), making it fast and cheap.
 
 ## Standardized
 
-Archethic's tokens are also unified through a specification to help implementers, developers and users to have a clear understanding of their definition
+Archethic's tokens are also unified through specifications to help implementers, developers and users to have a clear understanding of their definition.
 
-For more details please take a look at the [AEIP-2](https://github.com/archethic-foundation/aeip/blob/main/AEIP-2.md)
+The complete specification is composed of many AEIPs:
+
+- [AEIP-02 - Tokens standard](https://github.com/archethic-foundation/aeip/blob/main/AEIP-02.md)
+- [AEIP-18 - Tokens resupply](https://github.com/archethic-foundation/aeip/blob/main/AEIP-18.md)
+- [AEIP-19 - Tokens minting's recipients](https://github.com/archethic-foundation/aeip/blob/main/AEIP-19.md)
 
 ## Creation
 
 In order to create a token, you have to:
-1. set the transaction's type to: "token"
-2. define in the transaction's data the token definition
+
+1. set the transaction's type to: `token`
+2. define in the transaction's content the token's json definition
+
+[The JSON Schema is available on GitHub](https://github.com/archethic-foundation/archethic-node/blob/develop/priv/json-schemas/token-core.json)
+
+:::note
+As everything on Archethic, it is a good practice to create a new transaction chain for this token only.
+:::
 
 ### Fungible
 
-Here an example of a fungible token: 
-```jsonc
+Example of a token that doesn't allow minting (may never be resupplied):
+
+```json
```

Comment:
Suggestion: you can use `jsonc` to accept comments in the json snippet

Reply:
there is maybe a config to update for this. I updated the config to support elixir highlight